<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>619</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Общий стиль для окна */
QMainWindow {
    background-color: #2c2f33;
    color: #ffffff;
    font-family: Arial, sans-serif;
    font-size: 14px;
}

/* QGraphicsView стиль */
QGraphicsView {
    border: 2px solid #7289da;
    border-radius: 5px;
    background-color: #23272a;
}

/* QComboBox стиль */
QComboBox {
    border: 1px solid #99aab5;
    border-radius: 5px;
    padding: 5px;
    background-color: #23272a;
    color: #ffffff;
}
QComboBox::drop-down {
    border: none;
    background: #7289da;
    width: 20px;
}
QComboBox::down-arrow {
    image: url(:/icons/arrow-down.png); /* При необходимости замените иконку */
    width: 10px;
    height: 10px;
}

/* QPushButton стиль */
QPushButton {
    background-color: #7289da;
    border: 1px solid #7289da;
    border-radius: 5px;
    padding: 5px;
    color: #ffffff;
}
QPushButton:hover {
    background-color: #99aab5;
    border: 1px solid #99aab5;
}
QPushButton:pressed {
    background-color: #23272a;
    border: 1px solid #99aab5;
}

/* QSpacerItem стиль (используется для регулирования пространства) */
QSpacerItem {
    background: transparent;
}

/* ScrollBars (если они есть внутри QGraphicsView) */
QScrollBar:vertical {
    background: #2c2f33;
    border: none;
    width: 10px;
}
QScrollBar::handle:vertical {
    background: #7289da;
    border-radius: 5px;
}
QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical {
    background: none;
}
QScrollBar:horizontal {
    background: #2c2f33;
    border: none;
    height: 10px;
}
QScrollBar::handle:horizontal {
    background: #7289da;
    border-radius: 5px;
}
QScrollBar::add-line:horizontal,
QScrollBar::sub-line:horizontal {
    background: none;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Policy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="accountsCbx">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>35</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="chooseAccountBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>CHOOSE ACCOUNT</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>100</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="loadBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>LOAD LOG FILE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addAccountBtn">
        <property name="text">
         <string>ADD ACCOUNT</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
